includes/core/classes/settings/class-base.php

```diff
@@ -121,6 +121,7 @@ protected function get_sections(): array {
 	 * @return void
 	 */
 	protected function setup_hooks(): void {
+		add_action( 'init', array( $this, 'set_main_sub_page' ) );
```

Comment:
Why did you add this and the empty callbacks?

Reply:
@mauteri Fixed